config/src/config/vm_config.rs

```diff
@@ -1,119 +1,16 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use libra_types::transaction::SCRIPT_HASH_LENGTH;
-use serde::{Deserialize, Deserializer, Serialize, Serializer};
-use std::{collections::HashSet, hash::BuildHasher};
+use serde::{Deserialize, Serialize};
 
 /// Holds the VM configuration, currently this is only the publishing options for scripts and
 /// modules, but in the future this may need to be expanded to hold more information.
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 #[serde(default, deny_unknown_fields)]
-pub struct VMConfig {
-    pub publishing_options: VMPublishingOption,
-}
+pub struct VMConfig {}
```

Comment:
- Can we kill this struct altogether now?
- There are a couple of toml files in `test_data` that have `publishing_options`--can those sections be deleted now as well?